app/controllers/folders_controller.rb

```diff
@@ -16,11 +16,7 @@ class FoldersController < ApplicationController
   before_action :admin_user, only: [:new, :create, :edit, :update, :destroy]
 
   def index
-    @folders = if current_user.dl_locked_resources
-      Folder.order(:name)
-    else
-      Folder.where(locked: FALSE).order(:name)
```

Comment:
this is never hit because the before_action on line 15 only allows current_user.dl_locked_resources into the action